invenio_rdm_records/theme/views.py

```diff
@@ -120,3 +121,18 @@ def doi_identifier(pids):
 def doi_locally_managed(pid):
     """Determine if DOI is managed locally."""
     return is_doi_locally_managed(pid)
+
+
+@blueprint.app_template_filter('display_resource_type')
+def display_resource_type(resource_type):
+    """Returns formatted resource type string."""
+    vocabulary = Vocabulary.get_vocabulary('resource_types')
+    entry = vocabulary.get_by_dict(resource_type)
+
+    # NOTE: translations could also be done via the CSV file directly
+    result = _(entry.get('type_name'))
+    if entry.get('subtype_name'):
```

Comment:
Doubt: If (from last message IRL) we make mandatory subtype, it would always exist. Therefore, there is no need for an `if`. No?

Reply:
My understanding was that: we made the subtype mandatory last week and then yesterday during the call we reversed that decision because Zenodo has types without subtypes (e.g. poster). When @lnielsen takes a look he can clarify.

Reply:
Ah, I understood the decision was reverted once more after your metadata committee hehe sorry for the confusion.